src/header-utilities/is_container.hpp

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
I didn't intend for the comments to describe how things work. This is an area where I'd like a short introduction, or at least a description of how this particular example works.